packages/material-ui/src/Select/Select.test.js

```diff
@@ -671,6 +671,44 @@ describe('<Select />', () => {
 
       expect(getByRole('listbox')).to.be.ok;
     });
+
+    it('open only with the left mouse button click', () => {
+      // Test for https://github.com/mui-org/material-ui/issues/19250#issuecomment-578620934
+      // Right/middle mouse click shouldn't open the select
+      const { getByRole, queryByRole, getByTestId } = render(
+        <Select
+          MenuProps={{
+            transitionDuration: 0,
+            BackdropProps: { 'data-testid': 'select-backdrop' },
+          }}
+          value=""
+        >
+          <MenuItem value="">
+            <em>None</em>
+          </MenuItem>
+          <MenuItem value={10}>Ten</MenuItem>
+          <MenuItem value={20}>Twenty</MenuItem>
+          <MenuItem value={30}>Thirty</MenuItem>
+        </Select>,
+      );
+
+      const trigger = getByRole('button');
+
+      // Click by the left mouse button. Options list should be present
```

Comment:
This is already tested. It should be enough to just test if mouseDowns with other buttons don't open.